packages/whats-new/src/whats-new-page.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, GuidePage } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+
+function WhatsNewPage( { description, heading, imageSrc, isLastPage, key, link, pageNumber } ) {
+	const __ = useI18n().__;
+
+	useEffect( () => {
+		recordTracksEvent( 'wpcom_whats_new_slide_view', {
+			slide_number: pageNumber,
+			is_last_slide: isLastPage,
+		} );
+	}, [ isLastPage, pageNumber ] );
+
+	return (
+		<GuidePage className="whats-new-page__container">
+			<div className="whats-new-page__text">
+				{ heading && <h1 className="whats-new-page__heading">{ heading }</h1> }
+				<div className="whats-new-page__description">
+					{ description && <p>{ description }</p> }
+					{ link && (
+						<Button
+							className="whats-new-page__link"
+							href={ link }
+							isTertiary
+							isLink
+							target="_blank"
+						>
+							{ __( 'Learn more', 'Whats New Announcements' ) }
```

Comment:
I think there was a misunderstanding about how to do the `__i18n_text_domain__` trick for i18n-ized packages. There's more information here: pbAok1-1Ao-p2

```suggestion
							{ __( 'Learn more', __i18n_text_domain__ ) }
```

Basically you'll want to apply the `__i18n_text_domain__ ` argument to each call to `__` like I did in this suggestion.